cockroachdb/templates/_helpers.tpl

```diff
@@ -96,15 +96,48 @@ Define the default values for the certificate selfSigner inputs
 {{- end -}}
 
 {{/*
-Define the cron schedules for certificate rotate jobs
+Define the cron schedules for certificate rotate jobs and converting from hours to valid cron string
```

Comment:
Add a comment here that we assume each month has 31 days and as a result the cron job may fire earlier than specified.

31*12 = 371 day, which is probably fine

Reply:
I have updated the comment. Also, I have improved the logic which creates a cron schedule like `0 0 */28 * *` which means the cron will run every 28 days.

Also, added the helm template tests for the verification